src/nix/describe_derivation.nix

```diff
@@ -10,12 +10,12 @@
 
 let
   nixpkgs = builtins.getFlake "nixpkgs";
-  lib = import ./lib.nix { inherit nixpkgs; };
+  lib = import <lib> { inherit nixpkgs; };
```

Comment:
Why did you need to change this?

Reply:
Distributing via crates.io doesn't currently allow resource files to be included with your program. As such we need to write lib.nix to disk somewhere (which we do, to a /tmp/ file). This file is then added to the NIX_PATH, hence we need `<lib>`.

Reply:
We could technically pass the path to the lib.nix file in another env-var, but NIX_PATH already exists, so we might as well make use  of it.